packages/frontend/web-ui/src/common/helpers/validateFormConfirmPassword.ts

```diff
@@ -5,12 +5,13 @@ import { PASSWORD_MIN_LENGTH } from './passwordMinLength';
  * @param {String} password
  * @param {String} confirmPassword
  */
+
 export const validateFormConfirmPassword = (
-  formChecked,
-  password,
-  confirmPassword,
+  formChecked: { confirmPassword: string },
```

Comment:
This type doesn't seem to be complete. We should reconsider this parameter when using typescript. Wouldn't it be cleaner to provide the error as a return type instead with the error messages?

Reply:
Done! I've applied differents changes on the code. Could you revise it?